services/migrations/gitea_uploader.go

```diff
@@ -462,15 +462,28 @@ func (g *GiteaLocalUploader) CreateComments(comments ...*base.Comment) error {
 		if comment.Updated.IsZero() {
 			comment.Updated = comment.Created
 		}
-
+		if comment.CommentType == 0 {
+			// if type field is missing, then assume a normal comment
+			comment.CommentType = int64(issues_model.CommentTypeComment)
```

Comment:
Hmm, I think it might be beneficial to extract the comment types into `modules`, and then to make the comment type directly a `CommentType` instead of `int64`.

Reply:
Indeed, another PR made could be to move `CommentType` into the `structs` package